### ISSUE TITLE
Allow the scan workload to have an end row 

### ISSUE THREAD

**Issue by ramkrish86** [brianfrankcooper/YCSB#402]
Currently in HBaseclient the scan type operation does not have an end row so every time based on the start row it ends up in a full table scan (from the given start row).  It would be better if we can allow the option to specify an end row at fixed interval from the start row like a range of 1000, 10000 etc. Not sure if this is easily feasbile based on the current row generation pattern of YCSB.

**Comment by cmccoy** [comment]
I agree this will be hard with the current row generation strategy, but a [PageFilter](https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/filter/PageFilter.html) would limit the result size.

**Comment by ramkrish86** [comment]
True. But introducing filter is having its code path.  So this purely to test a simple scan range case.

**Comment by cmccoy** [comment]
It does, but the current implementation is probably an unnecessary disadvantage to HBase.
Cassandra limits to `recordcount` records ([thrift](https://github.com/brianfrankcooper/YCSB/blob/master/cassandra/src/main/java/com/yahoo/ycsb/db/CassandraClient10.java#L345), [cql](https://github.com/brianfrankcooper/YCSB/blob/master/cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java#L284)). Same with [MongoDB](https://github.com/brianfrankcooper/YCSB/blob/master/mongodb/src/main/java/com/yahoo/ycsb/db/AsyncMongoDbClient.java#L380).

**Comment by ramkrish86** [comment]
I do think it is more of client side feature limitation.  In real cases some time you may or may not know end row.  My idea was to just have a feature in YCSB similar to PE tool.
